l10n_it_fatturapa_in/models/account.py

```diff
@@ -43,6 +43,55 @@ class AccountInvoice(models.Model):
     e_invoice_received_date = fields.Date(
         string='E-Bill Received Date')
 
+    @api.depends('invoice_line_ids.price_subtotal', 'tax_line_ids.amount',
+                 'tax_line_ids.amount_rounding', 'currency_id', 'company_id',
+                 'date_invoice', 'type', 'efatt_rounding')
+    def _compute_amount(self):
+        super(AccountInvoice, self)._compute_amount()
+        self.amount_total += self.efatt_rounding
```

Comment:
Non mi è chiaro il motivo per cui modificare il campo base `amount_total` (e mi sembra un po' rischioso), nè per modificare `invoice_line_move_line_get`.
L'approccio non era aggiungere una riga fattura?

Reply:
La modifica si rende necessaria nel caso di arrotondamento su ImportoTotale.

L'approccio pensato precedentemente, aggiunta di una riga fattura, in questo caso non funziona in quanto andrebbe a modificare anche l'imponibile fallendone la validazione di quest'ultimo. 

Ho utilizzato la stessa tecnica del modulo account_global_discount:

https://github.com/OCA/account-invoicing/blob/6bd179e7abe36ce028e8a67e12c7d2e8698dba60/account_global_discount/models/account_invoice.py#L98

Reply:
Magari allora esegui tutti questi ricalcoli solo quando necessario? Cioè solo quando `efatt_rounding` è diverso da 0, giusto?

Reply:
Tra l'altro: è necessario il ricalcolo che considera la valuta?

Reply:
@eLBati Modificato in modo da rifare i calcoli solo per `efatt_rounding != 0`. Il ricalcolo per la valuta si rende necessario in quanto si ha una variazione di `amount_total` dovuto a `efatt_rounding`